treeherder/push_health/push_health.py

```diff
@@ -57,37 +67,39 @@ def get_push_failures(push, option_map):
         test=None
     ).select_related(
         'job_log__job__job_type', 'job_log__job__machine_platform'
-    ).prefetch_related(
-        'job_log__job__text_log_step__errors'
     )
 
     # using a dict here to avoid duplicates due to multiple failure_lines for
     # each job.
     tests = {}
     for failure_line in new_failure_lines:
         test_name = clean_test(failure_line.test)
-        test_key = '{}{}'.format(test_name, failure_line.job_guid)
+        job = failure_line.job_log.job
+        config = clean_config(option_map[job.option_collection_hash])
+        platform = clean_platform(job.machine_platform.platform)
+        jobName = job.job_type.name
+        jobSymbol = job.job_type.symbol
+        test_key = '{}{}{}{}'.format(test_name, config, platform, jobName)
+
         if test_name and test_key not in tests:
-            job = failure_line.job_log.job
-            config = option_map[job.option_collection_hash]
-            errors = []
-            for step in failure_line.job_log.job.text_log_step.all():
-                for error in step.errors.all():
-                    if len(errors) < 5:
-                        errors.append(error.line)
             line = {
                 'testName': test_name,
-                'logLines': errors,
-                'jobName': job.job_type.name,
-                'jobId': job.id,
-                'jobClassificationId': job.failure_classification_id,
-                'platform': job.machine_platform.platform,
-                'suggestedClassification': 'New Failure',
+                'jobName': jobName,
+                'jobSymbol': jobSymbol,
+                'platform': platform,
                 'config': config,
                 'key': test_key,
+                'jobs': [],
```

Comment:
This is where we keep track of multiple jobs where the platform, config, job_type, etc all match up.